dojo/filters.py

```diff
@@ -1366,7 +1366,7 @@ def __init__(self, *args, **kwargs):
                 Q(engagement__product__authorized_users__in=[get_current_user()]) |
                 Q(engagement__product__prod_type__authorized_users__in=[get_current_user()]))
             self.form.fields[
-                'duplicate_finding'].queryset = Finding.objects.filter(
+                'duplicate'].queryset = Finding.objects.filter(
```

Comment:
the problem is that `duplicate_finding` is excluded from the field list. so this whole statement with assigning a queryset can be removed. the `duplicate` field is a boolean so I think assigning a queryset that to a boolean field will either do nothing or raise an error.

Reply:
OH makes sense. Updating.